package.json

```diff
@@ -12,12 +12,18 @@
     "pretest": "lerna run pretest",
     "test": "node scripts/test.js",
     "posttest": "lerna run posttest",
-    "test:watch": "node scripts/test.js --watch",
-    "lint": "tsc -b && yarn prettier:check && eslint --ext .js,.jsx,.ts,.tsx,.graphql .",
-    "lint:fix": "eslint --ext .js,.jsx,.ts,.tsx,.graphql . --fix; prettier --ignore-path .eslintignore --write '**/*.{js,jsx,ts,tsx,graphql,md}'",
-    "prettier:check": "prettier --ignore-path .eslintignore --check '**/*.{js,jsx,ts,tsx,graphql,md}'",
-    "depcheck": "lerna exec --stream 'yarn depcheck --ignores=\"@app/config,@app/client,tslib,webpack,babel-plugin-import,source-map-support,@graphql-codegen/*,graphql-toolkit,net,tls,dayjs\" --ignore-dirs=\".next\"'",
-    "dev": "yarn && lerna run codegen --stream && tsc -b && concurrently --kill-others --names 'TSC,WATCH,RUN,TEST' --prefix '({name})' --prefix-colors 'yellow.bold,yellow.bold,cyan.bold,greenBright.bold' 'tsc -b --watch --preserveWatchOutput --assumeChangesOnlyAffectDirectDependencies' 'lerna run --parallel watch' 'lerna run --parallel dev' 'sleep 10; yarn test:watch'",
+    "test:watch": "run-script-os",
+    "test:watch:darwin:linux": "sleep 10 && yarn test:watch:default",
```

Comment:
I don't like the sleep being here; it should be in the calling location as if you want to just run `test:watch` locally (without `yarn start` running) you should be able to do so without incurring a 10 second delay. Is it necessary for it to have moved, or were you just tidying up?

Reply:
The delay is just so that the startup of the `dev` script doesn't get overwhelmed due to too much going on, it can get pretty intense for a few seconds. Maybe it would be better to pass an argument, e.g. `--delay 10`, to `scripts/test.js` from `concurrently` directly, and have the test script wait 10 seconds before starting. I'm assuming this has been done because Windows doesn't have a `sleep` command?

Reply:
Yes it was moved because Windows doesn't have a `sleep` command.  An JS timer would work as well, and is a bit cleaner from the `package.json` perspective.